README.md

```diff
@@ -8,26 +8,26 @@ The courtroom suite includes an abstract generic [SwearGame contract](#sweargame
 
 ### SwearGame contract
 
-This is the main general swear contract which conduct the trial.
+This is the main general swear contract which conducts the trial.
 
 It calls the specific [trial rules contract](#trial-rules-contract) to get the [witnesses](#witness-contract),get the trial statuses and transitions
 and to check for expiry time for a specific case and trial status.
 
-It calls the specific game witnesses contracts to testimony and validate for a submitted evidence.
+It calls the specific game witnesses contracts to testimony and validates for a submitted evidence.
 
 The SwearGame contract is aligned with the ABIs defined at SwearGameAbstract (courtroomabstract.sol).
 
 /// @notice () - a payable function which is used by the service as a deposit functionality
 
-///The function without name is the default function that is called whenever anyone sends funds to a contract
+///The function without name is the default function that is called when anyone sends funds to a contract
```

Comment:
I think you need to do some more reading:
http://solidity.readthedocs.io/en/develop/contracts.html#fallback-function